vars/multicloudDeploy.groovy

```diff
@@ -65,12 +66,21 @@ void call(parameters = [:]) {
 
             def deploymentType = DeploymentType.selectFor(CloudPlatform.CLOUD_FOUNDRY, config.enableZeroDowntimeDeployment).toString()
             def deployTool = script.commonPipelineEnvironment.configuration.isMta ? 'mtaDeployPlugin' : 'cf_native'
+            Boolean runInIsolatedWorkspace = false
+
+            if (config.cfTargets.size() > 1 && deploymentType == "blue-green") {
```

Comment:
please document why we need this, and what the condition says. Also I'm wondering if it would be more elegant to write 

```
bool runInIsolatedWorkspace = config.cfTargets.size() > 1 && deploymentType == "blue-green"
```

but that's minor.